<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QWidget" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>723</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>723</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="userlbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passlbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>731</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="userInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passInput">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="appPicture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>374</width>
     <height>58</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MINOR CINEPLEX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="guestButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login as Guest</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>userlbl</zorder>
  <zorder>passlbl</zorder>
  <zorder>userInput</zorder>
  <zorder>passInput</zorder>
  <zorder>appPicture</zorder>
  <zorder>loginButton</zorder>
  <zorder>guestButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
